pinot-query-runtime/src/test/resources/queries/Aggregates.json

```diff
@@ -164,85 +117,38 @@
         "comments": "class java.lang.Double cannot be cast to class java.lang.Boolean",
         "description": "aggregate boolean column",
         "sql": "SELECT min(bool_col) FROM {tbl}"
+      },
+      {
+        "psql": "9.21.0",
+        "description": "aggregate boolean column",
+        "sql": "SELECT bool_and(bool_col), bool_or(bool_col) FROM {tbl} GROUP BY string_col"
+      },
+      {
+        "psql": "9.21.0",
+        "description": "aggregate boolean column no group by",
+        "sql": "SELECT bool_and(bool_col), bool_or(bool_col) FROM {tbl}"
```

Comment:
do we support `SELECT bool_and(startsWith(string_col, 'foo')) FROM ...` ?
can we add a test for these type of use case?

Reply:
we do not support this yet (for any aggregate function in v2). This is a huge gap, good call out.